### ISSUE TITLE
Error: Cannot find module 'lodash/snakecase' when use  vue-cli-plugin-s3-deploy@next

### ISSUE THREAD

**Issue by jk2K** [multiplegeorges/vue-cli-plugin-s3-deploy#71]
```bash
$ vue-cli-service s3-deploy                                                                                                                                    internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'lodash/snakecase'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/mnt/e/dev/frontend/simplepaper-web/node_modules/vue-cli-plugin-s3-deploy/configuration.js:8:41)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
error Command failed with exit code 1.                                                                                                                         info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

and I cannot install  lodash/snakecase
```bash
 yarn add lodash/snakecase                                                                                                    yarn add v1.13.0                                                                                                                                               $ npm run npmcheckversion                                                                                                                                      npm WARN lifecycle The node binary used for scripts is /tmp/yarn--1561999084827-0.8815714044724983/node but npm is using /home/lee/.nvm/versions/node/v10.14.0/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> react-boilerplate@3.7.0 npmcheckversion /mnt/e/dev/frontend/simplepaper-web
> node ./internals/scripts/npmcheckversion.js

warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/5] Validating package.json...                                                                                                                               [2/5] Resolving packages...                                                                                                                                    warning Resolution field "babel-core@7.0.0-bridge.0" is incompatible with requested version "babel-core@^6.0.0"                                                warning Resolution field "babel-core@7.0.0-bridge.0" is incompatible with requested version "babel-core@^6.0.0"                                                error Command failed.                                                                                                                                          Exit code: 128
Command: git
Arguments: ls-remote --tags --heads ssh://git@github.com/lodash/snakecase.git
Directory: /mnt/e/dev/frontend/simplepaper-web
Output:
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command
```

**Comment by mhluska** [comment]
@jk2K replace this line in `src/configuration.js`:

```js
import snakeCase from 'lodash/snakecase'
```

With this:

```js
import { snakeCase } from 'lodash';
```

**Comment by nicekiwi** [comment]
fixed in c9c7fd123c2e4d5744aef86aef9489c35857e894
